dist/index.js

```diff
@@ -975,6 +975,21 @@ const exec = __importStar(__webpack_require__(986));
 class Ndk {
     constructor(path) {
         this.path = path;
+
+//        var OSName="Unknown OS";
```

Comment:
This PR is just a try out.

Here I run into 
<img width="418" alt="image" src="https://user-images.githubusercontent.com/3314607/79944287-6d662800-846b-11ea-9f7f-9a864f5cb1a4.png">

Reply:
the index.js file is auto-generated from the code `npm run all` and doesn't need to be updated manually.

Reply:
Good to know !
But why add this file to git, when it's auto-generated. Why not remove from git ?

Reply:
It seems the GHA directly uses dist folder: https://github.com/actions/typescript-action#publish-to-a-distribution-branch